getconf/base.py

```diff
@@ -146,8 +146,14 @@ def _get(self, env_key, section, key, default=''):
 
         return default
 
-    def get(self, key, default='', doc=''):
-        """Fetch a value from the various sources."""
+    def get(self, key, default='', doc='', check_default=True):
```

Comment:
Split this method into two separate methods, one for the deprecation part and one for this internal part.